src/filters/booksFilter.js

```diff
@@ -0,0 +1,5 @@
+const booksFilter = (bookArray, filterCategory) => (
+  filterCategory !== 'All' ? bookArray.filter(book => book.category === filterCategory) : bookArray
+);
+
+export default booksFilter;
```

Comment:
**OPTIONAL**: This is nice, I like what you did here, but not sure if having this on it's own module is adding mental tax, the maintainer (yourselves included) will need to figure out why is this here, what is it doing and where is it used.

Please take a look at this article https://kentcdodds.com/blog/colocation